tests/test_processing_common.py

```diff
@@ -88,6 +92,13 @@ def prepare_image_inputs(self):
         image_inputs = [Image.fromarray(np.moveaxis(x, 0, -1)) for x in image_inputs]
         return image_inputs
 
+    @require_vision
+    def prepare_video_inputs(self):
+        video_file = hf_hub_download(
+            repo_id="raushan-testing-hf/videos-test", filename="video_demo.npy", repo_type="dataset"
+        )
+        return [np.load(video_file)]
```

Comment:
Same as in other PR, no need to download. We can define random 4D array

Reply:
oh, I just copy-pasted this from the other PR

but none of the models here actually support videos so I'll remove this here for now & modify the other PR instead